src/BlazorWebView/src/Maui/build/Microsoft.AspNetCore.Components.WebView.Maui.targets

```diff
@@ -7,10 +7,8 @@
 
   <!-- Prevent Razor files from being packaged as bundle resources in iOS/MacCatalyst targets by declaring them
        as RazorComponent instead of Content. -->
-  <ItemGroup>
+  <ItemGroup Condition="'$(DesignTimeBuild)' != 'true'">
     <Content Remove="**\*.razor" />
-  </ItemGroup>
-  <ItemGroup>
```

Comment:
(These never needed to be two separate ItemGroups, and by combining them I don't have to repeat the new Condition. Refactoring XML is good.)